policy-controller/grpc/src/outbound.rs

```diff
@@ -529,3 +538,20 @@ fn default_queue_config() -> outbound::Queue {
         ),
     }
 }
+
+fn convert_duration(
+    name: &'static str,
+) -> impl Fn(time::Duration) -> Option<prost_types::Duration> {
+    move |duration| {
+        duration
+            .try_into()
+            .map_err(|error| {
+                tracing::error!(%error, "Invalid {name} duration");
+                prost_types::Duration {
```

Comment:
Just for my understanding, this value is irrelevant because of the subsequent call to `ok()`, right?  could this be `()`?

Reply:
ah, yup, this is an artifact of an earlier version where i had this return the maximum representable protobuf duration, before i realized that the existing code just ignores durations that are too big for protobuf. i'll remove the struct entirely.